ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-footer/gio-menu-footer.component.scss

```diff
@@ -17,13 +17,17 @@
 @use '@angular/material' as mat;
 @use '../../../scss' as gio;
 
+$background: map.get(gio.$mat-oem-palette, background);
+$text: map.get(gio.$mat-oem-palette, background-contrast);
+$menu-border: color-mix(in srgb, $background 50%, $text);
```

Comment:
Only the `menu-border` is used in this file.

Reply:
background and text are use in the color-mix